com.unity.netcode.gameobjects/Runtime/Transports/NetworkTransport.cs

```diff
@@ -1,163 +1,71 @@
 using System;
-using System.Collections.Generic;
 using UnityEngine;
 
 namespace Unity.Netcode
 {
-    public enum NetworkChannel : byte
-    {
-        Internal,
-        TimeSync,
-        ReliableRpc,
-        UnreliableRpc,
-        SyncChannel,
-        DefaultMessage,
-        PositionUpdate,
-        AnimationUpdate,
-        NavAgentState,
-        NavAgentCorrection,
-        NetworkVariable, //todo: this channel will be used for snapshotting and should then go from reliable to unreliable
-        SnapshotExchange,
-        Fragmented,
-        ChannelUnused, // <<-- must be present, and must be last
-    };
-
-    /// <summary>
-    /// A network transport
-    /// </summary>
     public abstract class NetworkTransport : MonoBehaviour
```

Comment:
there are lots of API breaking changes (for good) in this file. in fact, these changes are influencing changes in other places as well.

Reply:
I assume you and @andrews-unity are coordinating since he's got his transport refactor coming

Reply:
yep, we had a quick chat about this and he'll also review this PR too.